src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,57 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
-
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
-
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button
-          data-cy="clearSearchButton"
-          type="button"
-          className="delete"
+/* eslint-disable jsx-a11y/control-has-associated-label */
+interface TodoFilterProps {
+  onQueryChange: (event: React.ChangeEvent<HTMLInputElement>) => void
+  onFilterChange: (event: React.ChangeEvent<HTMLSelectElement>) => void
+  onDelete: () => void
+  query: string
+  filter: string
+}
+export const TodoFilter: React.FC<TodoFilterProps> = ({
+  onQueryChange,
+  onFilterChange,
+  onDelete,
+  query,
+  filter,
+}) => {
+  return (
```

Comment:
when you have no logic in the component and just return jsx, then you can delete the word `return` and return directly the jsx

`}) => (`